easybuild/easyconfigs/s/SoX/SoX-14.4.2-GCCcore-11.3.0.eb

```
@@ -0,0 +1,38 @@
+# Institution: IT4Innovations National Supercomputing Center, Czech Republic
+# Author: Jakub Kropacek
+# License: GPLv3
+# Year: 2022
+
+easyblock = 'ConfigureMake'
+
+name = 'SoX'
+version = '14.4.2'
+
+homepage = 'http://sox.sourceforge.net/'
+docurls = 'http://sox.sourceforge.net/Docs/Documentation'
+description = """Sound eXchange, the Swiss Army knife of audio manipulation"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://sourceforge.net/projects/sox/files/sox/%(version)s']
```

Sorry for the nitpicking, but before approving this I did a diff with the existing easyconfig, and found that you changed `SOURCEFORGE_SOURCE` into this URL. It's a bit cleaner and shorter to use the template variable (and it still seems to work for me), so could we change this back?
```suggestion
source_urls = [SOURCEFORGE_SOURCE]
```

---

No problem, it does indeed work and I agree with your points.